README.adoc

```diff
@@ -36,30 +38,238 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 
 [[initial]]
-== Create a resource controller
+== Add required dependencies
```

Comment:
Is the current version good enough? The text was removed on a later revision, besides the Alpha reminder which I think is relevant.